rxmongo/src/main/scala/akka/contrib/persistence/mongodb/RxMongoReadJournaller.scala

```diff
@@ -123,7 +127,8 @@ class CurrentAllEvents(val driver: RxMongoDriver) extends IterateeActorPublisher
     journal.find(BSONDocument())
       .projection(BSONDocument(EVENTS -> 1))
       .cursor[BSONDocument]()
-      .enumerate()
+      .enumerator()
+      .map(doc => doc) // this is needed for suffix collections
```

Comment:
Done !

Well, this PR should be renamed: "just add a test to really check CurrentAllEvents behavior with suffixed collections" ;-)